scripts/post_deploy_tdr.py

```diff
@@ -64,6 +72,11 @@ def verify_source(source_spec: TDRSourceSpec):
     require(source.location.lower() == config.tdr_source_location.lower(),
             'Actual storage location of TDR source differs from configured one',
             source.location, config.tdr_source_location)
+    ref = TDRSourceRef(id=source.id, spec=source_spec)
```

Comment:
I'm surprised `tdr.lookup_source` doesn't return a SourceRef instance but an instance of some one-off class. Do you know why?

Reply:
Because `SourceRef` lacks the `region` field.

Reply:
Could you add FIXMEs referring to https://github.com/DataBiosphere/azul/issues/5524 here and above azul.terra.TDRClient.TDRSource.